### PR TITLE
README reminders + roadmap package created

### PR DESCRIPTION
**Pull request by MedericFourmy**

### CODE REVIEW

File: ProjetSupaero2018/catkin_ws/src/roadmap/package.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <!-- Use buildtool_depend for build tool packages: -->
+  <!--   <buildtool_depend>catkin</buildtool_depend> -->
+  <!-- Use exec_depend for packages you need at runtime: -->
+  <!--   <exec_depend>message_runtime</exec_depend> -->
+  <!-- Use test_depend for packages you need only for testing: -->
+  <!--   <test_depend>gtest</test_depend> -->
+  <!-- Use doc_depend for packages you need only for building documentation: -->
+  <!--   <doc_depend>doxygen</doc_depend> -->
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
```
Comment (martinRenou) [inline ProjetSupaero2018/catkin_ws/src/roadmap/package.xml]: I'm not sure, do we need those dependencies ? What are they for ?

Comment (martinRenou) [reply ProjetSupaero2018/catkin_ws/src/roadmap/package.xml]: std_msgs rospy and roscpp ?

Comment (MedericFourmy) [reply ProjetSupaero2018/catkin_ws/src/roadmap/package.xml]: roscpp might not be necessary indeed, I can remove it if you want

Comment (martinRenou) [reply ProjetSupaero2018/catkin_ws/src/roadmap/package.xml]: As you wish, if it's not that important you can leave it as is

**Pull request merged**